snap/snapcraft.yaml

```diff
@@ -6,7 +6,7 @@ description: |
   console. The only difference is that Newsboat is actively maintained 
   while Newsbeuter isn't.
 grade: stable
-confinement: strict
```

Comment:
https://docs.snapcraft.io/reference/confinement says "strict" is the default and is recommended. https://snapcraft.io/blog/demystifying-snap-confinement says "classic" should be used only as a stopgap for situations where plugs aren't enough to make the snap functional. I'm pretty sure #49 was caused by user error; did you run into the same issue and have more details?